wasm/jit/jit_value_location_amd64.go

```diff
@@ -104,9 +104,10 @@ func newValueLocationStack() *valueLocationStack {
 }
 
 type valueLocationStack struct {
```

Comment:
I think I should add comments on this struct

Reply:
are you expecting me to say "no you shouldn't!"?😎

Reply:
https://github.com/tetratelabs/wazero/pull/68/commits/86b0105859364caa0340ed61a82a99c557923cb7 here you go!